clang/lib/CIR/CodeGen/CIRGenBuiltinX86.cpp

```diff
@@ -18,9 +18,22 @@
 #include "clang/CIR/MissingFeatures.h"
 #include "llvm/IR/IntrinsicsX86.h"
 
+#define UNIMPLEMENTED_BUILTIN()                                                \
+  do {                                                                         \
+    cgm.errorNYI(e->getSourceRange(),                                          \
+                 std::string("unimplemented X86 builtin call: ") +             \
+                     getContext().BuiltinInfo.getName(builtinID));             \
+    return {};                                                                 \
+  } while (0)
+
```

Comment:
I don't really like this change. It hides the return and searching for the source of the NYI diagnostic no longer takes you directly to where the code needs to be added. With this being temporary code, the verbose repetition in situ is preferable.